test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.cs

```diff
@@ -73,6 +80,8 @@ public void P2PDifferentFrameworks(DnxSdk sdk)
 
             // Assert
             Assert.Equal(0, result.ExitCode);
+
+            TestUtils.CleanUpTestDir<DnuPackTests>(sdk);
```

Comment:
The issue with the using pattern is that dispose is called at the end of the test regardless of whether the test passed or failed. In fact, there isn't a good way of detecting this as part of xUnit. Hence the not so ideal way of cleaning up.